Packs/Tenable_io/Integrations/TenableioEventCollector/TenableioEventCollector.py

```diff
@@ -0,0 +1,446 @@
+    if status == 'FINISHED':
+        for chunk_id in chunks_available:
+            vulnerabilities.extend(client.download_vulnerabilities_chunk(export_uuid=export_uuid, chunk_id=chunk_id))
+        readable_output = tableToMarkdown('Vulnerabilities List:', vulnerabilities,
+                                          removeNull=True,
+                                          headerTransform=string_to_table_header)
+
+        results = CommandResults(readable_output=readable_output,
+                                 raw_response=vulnerabilities)
+        return PollResult(response=results)
+    elif status in ['CANCELLED', 'ERROR']:
+        results = CommandResults(readable_output='Export job failed',
+                                 entry_type=entryTypes['error'])
+        return PollResult(response=results)
+    else:
+        results = CommandResults(readable_output='Export job failed',
+                                 entry_type=entryTypes['error'])
+        return PollResult(continue_to_poll=True, args_for_next_run={"export_uuid": export_uuid, **args},
+                          response=results)
+
+
+''' FETCH COMMANDS '''
+
+
+def fetch_vulnerabilities(client: Client, last_run: dict, severity: List[str]):
+    """
+    Fetches vulnerabilities if job has succeeded.
+    Args:
+        last_run: last run object.
+        severity: severity of the vulnerabilities to include in the export.
+        client: Client class object.
+
+    Returns:
+        Vulnerabilities fetched from the API.
+    """
+    vulnerabilities = []
+    export_uuid = last_run.get('export_uuid')
+    last_found_use = last_run.get('last_found_use')  # last run fetch time
+    if export_uuid:
+        demisto.info(f'Got export uuid from API {export_uuid}')
+        vulnerabilities, status = try_get_chunks(client=client, export_uuid=export_uuid)
+        # set params for next run
+        if status == 'FINISHED':
+            last_run.update({'export_uuid': None})
+        elif status in ['CANCELLED', 'ERROR'] and last_found_use:
+            export_uuid = client.get_export_uuid(num_assets=5000, last_found=last_found_use, severity=severity)
+            last_run.update({'export_uuid': export_uuid})
+
+    demisto.info(f'Done fetching {len(vulnerabilities)} vulnerabilities, Setting {last_run=}.')
+    return vulnerabilities
+
+
+def fetch_events_command(client: Client, first_fetch: Optional[datetime], last_run: dict, limit: int = 1000):
+    """
+    Fetches audit logs.
+    Args:
+        client: Client class object.
+        first_fetch: time to first fetch from.
+        last_run: last run object.
+        limit: number of audit logs to max fetch.
+
+    Returns: vulnerabilities, audit logs and updated last run object
+
+    """
+
+    last_fetch = last_run.get('next_fetch')
+    last_index_fetched = last_run.get('index_audit_logs')
+    index = int(last_index_fetched) if last_index_fetched is not None else 0
```

Comment:
```suggestion
    last_index_fetched = last_run.get('index_audit_logs', 0)
```
do we need to cast to int?